qiskit_ibm/jupyter/jobs_widget.py

```diff
@@ -253,7 +252,7 @@ def callback(trace, selection, _):  # pylint: disable=unused-argument
     return sun_wid
 
 
-def jobs_tab(backend: Union[IBMQBackend, FakeBackend]) -> wid.HBox:
+def jobs_tab(backend: Union[IBMQBackendService, FakeBackend]) -> wid.HBox:
```

Comment:
I don't think this should be changed to IBMQBackendService. `jobs_tab` is being called from backend_info.py and a IBMQBackend is passed to it. Wouldn't that fail if this is changed to `IBMQBackendService`? I think we should still keep it IBMQBackend and then do:
```
provider = backend.provider()
jobs = provider.backend.jobs(...)
```
and then fix the test cases accordingly.